### ISSUE TITLE
Priority support 

### ISSUE THREAD

**Issue by amomagus** [WemXPro/tracker#390]
### Describe the solution you'd like

it’s a system where you can give groups/ users a special priority like Highest/High/Medium/Low/lowest and their ticket would display which priority they have (as icon with text) and they would automatically (by default) be listed by prority  you also have to give users permission to set their own priority in the ticket and while creating a ticketing there is also a permission to set it automatically or per package (so if user owns „example“ they can set their own level to highest or the level will automatically be highest

**Comment by VanillaChan6571** [comment]
Easily can be abused, work around is having departments as some type of “higher” priority.

**Comment by amomagus** [comment]
Yeah, but you can’t like set it were a user owns a package the department will be assigned. Right?
